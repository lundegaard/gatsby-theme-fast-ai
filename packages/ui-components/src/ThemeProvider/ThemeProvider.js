import React from 'react';
import { ThemeProvider as EmotionProvider } from 'emotion-theming';

const ThemeProvider = props => <EmotionProvider {...props} />;

export default ThemeProvider;
