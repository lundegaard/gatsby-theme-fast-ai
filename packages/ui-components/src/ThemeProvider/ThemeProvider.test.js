import React from 'react';
import { shallow } from 'enzyme';

import ThemeProvider from './ThemeProvider';

describe('<ThemeProvider />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<ThemeProvider {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
