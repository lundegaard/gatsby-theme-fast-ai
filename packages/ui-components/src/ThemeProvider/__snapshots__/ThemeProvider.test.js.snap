// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ThemeProvider /> matches snapshot when shallowly rendered 1`] = `<ThemeProvider />`;
